### DISCUSSION TITLE
Autocomplete for Go stops working after some time

### DISCUSSION THREAD

**Discussion by akoskm** [LazyVim/LazyVim#2383] [Q&A]
What you can see on the attached video is basically:

1. Autocomplete not working
2. Restarting Vim and restoring the session
3. Typing the same thing where the autocomplete previously didn't work
4. Now it works

https://github.com/LazyVim/LazyVim/assets/3111627/d987fff8-3e91-4ef4-9fa7-9786926368b0

**Comment by 0x2E** [comment]
It may be an LSP bug. I've encountered this issue countless times, in both VSCode an Neovim.

Restarting LSP always works for me, I did:

- nvim: `:LspRestart`
- VSCode: cmd+shift+p and `Reload Windows` command
